flake.nix

```diff
@@ -240,12 +240,18 @@
           pkgsUnstable = import inputs.nixpkgs-unstable { inherit system; };
 
           gitRev = if (builtins.hasAttr "rev" self) then self.rev else "dirty";
+          gitShortRev = self.shortRev or (self.dirtyShortRev or "dirty");
+          lastModified = if self ? lastModified then builtins.toString self.lastModified else "0";
+          lastModifiedDate = self.lastModifiedDate or "1970-01-01T00:00:00Z";
```

Comment:
Additional flake metadata for embedding in `app2` builds.

@benluelo I can not cast `lastModified` (normally an integer representing seconds) if you would like to use this elsewhere.

Reply:
wdym you can't cast it?

Reply:
I can, just casted here because lazy. Eh, I'll adjust to integer anyway so it's an expected result and cast to string only in the `app2` derivation.